go/arrow/csv/writer_test.go

```diff
@@ -51,25 +52,111 @@ func TestCSVWriter(t *testing.T) {
 	rec := b.NewRecord()
 	defer rec.Release()
 
-	w := csv.NewWriter(f, schema, csv.WithFieldSeparator(';'))
+	w := csv.NewWriter(f, schema, csv.WithComma(';'))
+	err := w.Write(rec)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	r := csv.NewReader(f, schema, csv.WithComment('#'), csv.WithComma(';'))
+	defer r.Release()
+
+	n := 0
+	for r.Next() {
+		rec := r.Record()
+		for i, col := range rec.Columns() {
+			fmt.Printf("rec[%d][%q]: %v\n", i, rec.ColumnName(i), col)
```

Comment:
`s/i,/n,/`

Reply:
i is column index, n is row index, here it should be i, right?

Reply:
if this is wrong, then we need update read_test.go as well. : )

Reply:
yep, it should print:
```
rec[0]["i64"]: [0]
rec[0]["f64"]: [0]
rec[0]["str"]: ["str-0"]
rec[1]["i64"]: [1]
rec[1]["f64"]: [1]
rec[1]["str"]: ["str-1"]
```